controllers/velero.go

```diff
@@ -611,3 +617,39 @@ func (r *DPAReconciler) getResticResourceReqs(dpa *oadpv1alpha1.DataProtectionAp
 
 	return ResourcesReqs
 }
+
+func (r DPAReconciler) noDefaultCredentials(dpa oadpv1alpha1.DataProtectionApplication) (map[string]bool, bool, error) {
```

Comment:
A function comment on return vals would be nice so you don't have to inspect the function for what is returned.